inference-engine/src/cldnn_engine/cldnn_executable_network.cpp

```diff
@@ -95,7 +95,7 @@ IInferRequestInternal::Ptr CLDNNExecNetwork::CreateInferRequest() {
     return CreateAsyncInferRequestFromSync<CLDNNAsyncInferRequest>();
 }
 
-InferenceEngine::CNNNetwork CLDNNExecNetwork::GetExecGraphInfo() {
+std::shared_ptr<ngraph::Function> CLDNNExecNetwork::GetExecGraphInfo() {
```

Comment:
Maybe in order to avoid immediately changes for all plugins, we can continue to return CNNNetwork for `ExecGraphInfo` and for new API just return `CNNNetwork().getFucntion()` or it doesn't work?

Reply:
All plugins produce ngraph::Function. I think It is the right time to change this internal API.